PSNeo4j/Public/Invoke-Neo4jQuery.ps1

```diff
@@ -124,6 +128,12 @@
                 if($PSBoundParameters.ContainsKey('Parameters')) {
                     Add-Member -InputObject $Statement -Name 'parameters' -Value $Parameters -MemberType NoteProperty
                 }
+                if($AsGraph.IsPresent) {
+                    Add-Member -InputObject $Statement -Name 'resultDataContents' -Value ([string[]]('graph')) -MemberType NoteProperty
+                    if ($As -notin 'Raw','Graph') {
```

Comment:
Hi!

I do like keeping the `AsGraph` switch (lets you use Raw and keep that data), but would it be worth doing something like this?

```powershell
if (-not $PSBoundParameters.ContainsKey(As)) {
    $As = 'Graph'
}
```

This way if the user doesn't explicitly set `As`, they can simply use `AsGraph` and get results that they would expect (a graph)?

If you'd rather not do that now, I can try adding it later, your call - Thanks!

Warren

Reply:
Sounds clever, I will get on it right now.